core/src/main/java/org/apache/druid/java/util/common/parsers/JSONPathFieldSpec.java

```diff
@@ -23,32 +23,55 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Preconditions;
 
+import java.util.List;
 import java.util.Objects;
 
 public class JSONPathFieldSpec
 {
   private final JSONPathFieldType type;
   private final String name;
   private final String expr;
+  private final List<String> exprs;
 
   @JsonCreator
   public JSONPathFieldSpec(
       @JsonProperty("type") JSONPathFieldType type,
       @JsonProperty("name") String name,
-      @JsonProperty("expr") String expr
+      @JsonProperty("expr") String expr,
+      @JsonProperty("exprs") List<String> exprs
```

Comment:
`exprs` and `expr` may be ambiguous. I think maybe we can name it as `nodes` because node is a concept related to a tree.

Reply:
fixed in 672e039e519728bde493bf2ab076df9d586db4f9.